### PR TITLE
Instrument Icons: UI changes

### PR DESCRIPTION
**Pull request by Riley-I**
**Short description of changes**

**Context: Fixes an issue?**

**Does this change need documentation? What needs to be documented and how?**

**Status of this Pull Request**

**What is missing until this pull request can be merged?**

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

**Comment by Riley-I** [comment]
As discussed, I have replaced 3 instrument icons for testing (violin, djembe, flute) in the following directory: https://github.com/jamulussoftware/jamulus/tree/master/src/res/instruments

These are my first drafts of a possible style of icon (based off of the current visual style/identity of the software and website). Please also commend and provide feedback based on image attached. 

***Note: the light, dark, and blue versions for the light and dark mode respectively.
Jamulus-Icons

**Review by ann0see — COMMENTED** [review]
I think the size of the doesn’t match yet.

**Comment by ann0see** [comment]
No need to open a new PR. Just push to the branch you were using for this one

**Comment by ann0see** [comment]
Ok. See 
image

I still have the feeling it doesn't fit (contrast?)

**Comment by Riley-I** [comment]
@ann0see do you think the contrast would be better if the icon was white or black?

Should I upload both a dark and a light version?

**Comment by Riley-I** [comment]
@ann0see I've re-uploaded the files but in dark. let me know how the contrast looks
https://github.com/Riley-I/jamulus/tree/autobuild/UIChanges/src/res/instruments

**Comment by ann0see** [comment]
You can download the app yourself here: https://github.com/Riley-I/jamulus/actions I'll take a screenshot asap.

**Comment by ann0see** [comment]
grafik

and fancy skin:

grafik


I think that's better. Maybe the image could be a bit bigger.

File: src/res/instruments/djembe.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19 27"><defs><style>.cls-1{fill:#242323;}</style></defs><path class="cls-1" d="M13.55,15.19h0l2.33-8.7-.14-.63a.58.58,0,0,0,0-.19c0-1.27-2.8-2.3-6.26-2.3-3,0-5.49.77-6.1,1.81h0v0a.92.92,0,0,0-.14.47s0,.08,0,.12l-.15.7L6,15.1c0,.05.06.19.48.5l-1.34,6a6,6,0,0,0,4.42,2,8.87,8.87,0,0,0,4.83-1.5L13,15.71Zm-.5-6.36a20.36,20.36,0,0,1-1.14,5.34c-.1.29-.57.17-.46-.12a19.86,19.86,0,0,0,1.12-5.22C12.59,8.52,13.07,8.52,13.05,8.83ZM9.59,3.69c2.9,0,5.25.56,5.25,1.26S12.49,6.21,9.59,6.21,4.34,5.64,4.34,5,6.69,3.69,9.59,3.69Zm-5,3.22a11.73,11.73,0,0,0,2,.47,22.69,22.69,0,0,0,8-.47c.31-.06.43.4.13.47-.76.15-1.52.29-2.28.4A22.9,22.9,0,0,1,8.26,8a13.76,13.76,0,0,1-3.82-.63C4.14,7.28,4.27,6.81,4.56,6.91Zm5.3,7.6a.24.24,0,0,1-.48,0,35.36,35.36,0,0,1-.21-5.44c0-.31.49-.31.48,0A35.36,35.36,0,0,0,9.86,14.51Zm-3.09-.24a37.23,37.23,0,0,1-.92-5.44c0-.31.46-.31.49,0a35.46,35.46,0,0,0,.89,5.31C7.31,14.44,6.84,14.57,6.77,14.27Zm5.47,1.79a16.17,16.17,0,0,1-2.59.29,7.86,7.86,0,0,1-2.37-.23.24.24,0,0,1,.13-.46,7.35,7.35,0,0,0,2.24.21,14.48,14.48,0,0,0,2.46-.28C12.41,15.54,12.54,16,12.24,16.06Z"/></svg>
```
Comment (pljones) [inline src/res/instruments/djembe.svg:1]: Do we have a style guide for XML-ish files?  "Pretty" XML, plus a copyright header?
```
<!-- Copyright YYYY Jamulus Development team
        author: Riley-I (https://github.com/Riley-I) -->
```

**Comment by Riley-I** [comment]
> grafik
> 
> and fancy skin:
> 
> grafik
> 
> I think that's better. Maybe the image could be a bit bigger.

Just uploaded files 10px bigger width and height.

**Comment by ann0see** [comment]
Great. Thanks. Sorry for the delay, I think it looks quite good now:
grafik

**Comment by Riley-I** [comment]
I'll be making more icon drafts this weekend. Should I just keep using this PR to upload them as I create them?

It will be quite an agile/iterative process so there will be lots of changes.

**Comment by ann0see** [comment]
I mean this PR doesn't harm (as long as it's not open for ages, but we're having that already for other PRs). You can still close it and comment your process on the thread you opened?

**Comment by Riley-I** [comment]
> 

ok sounds good:). I'll post my progress. Yeah it makes sense to close it as the PR for the initial drafts - I will do the rest of the icons in batches of maybe 5 icons at a time and post them for feedback as I complete them.

**Comment by ann0see** [comment]
Yeah. Ok. I hope this will get more attention then. Closing this PR.

File: src/res/instruments/djembe.svg (hunk 1 above)
Comment (ann0see) [reply src/res/instruments/djembe.svg]: Ah yes, please keep this in mind if you add svg files.

**Pull request closed without merging**